packages/modules/src/utils/Helper.ts

```diff
@@ -39,3 +52,23 @@ export const getUserOpHash = (userOp: Partial<UserOperationStruct>, entryPointAd
   const enc = encodeAbiParameters(parseAbiParameters("bytes32, address, uint256"), [userOpHash, entryPointAddress, BigInt(chainId)]);
   return keccak256(enc);
 };
+
+export async function getABISVMSessionKeyData(sessionKey: `0x${string}` | Uint8Array, permission: Permission): Promise<`0x${string}` | Uint8Array> {
```

Comment:
wouldn't this be dynamic? how are we making easy here to have any types of rules / conditions